awx/main/scheduler/kubernetes.py

```diff
@@ -51,6 +52,43 @@ def list_active_jobs(self, instance_group):
 
         return pods
 
+    @classmethod
+    def create_secret(self, instance_group, credential):
```

Comment:
Specifically... this has to be a registry credential?

Reply:
Yeah, it might be good to either docstring that info, or give it some more specific naming. This is really only for registry credentials, and that's not natural to follow with how you've named things so far.